pyface/tasks/i_task_window_backend.py

```diff
@@ -31,6 +31,18 @@ def destroy(self):
         but this method may be used to perform additional cleanup.
         """
 
+    def begin_batch_window_updates(self):
```

Comment:
Note hugely critical, but a context manager that calls this method on `__enter__` and the corresponding end method `__exit__` would make the Wx code a bit more robust, I think.